packages/sdk/typescript/human-protocol-sdk/src/graphql/queries/operator.ts

```diff
@@ -19,7 +19,6 @@ const LEADER_FRAGMENT = gql`
     publicKey
     webhookUrl
     url
-    jobTypes
```

Comment:
Throwing error: jobTypes does not exists. 
Same list of fields here `python/human-protocol-sdk/human_protocol_sdk/gql/operator.py`.
Is it wrong?

Reply:
It does exist. Something else might be wrong. Have in mind it might be empty.
If job types is not on python, should be added
![image](https://github.com/humanprotocol/human-protocol/assets/61605646/c894a93f-6cf9-41c7-b313-4a894f3554cb)